src/LessMsi.Cli/Program.cs

```diff
@@ -188,10 +192,23 @@ private static void PrintProgress(IAsyncResult result)
 
         private static void EnsureFileRooted(ref string sFileName)
         {
-            if (!Path.IsPathRooted(sFileName))
+            if (!IsPathRooted(sFileName))
             {
                 sFileName = Path.Combine(Directory.GetCurrentDirectory(), sFileName);
             }
         }
+
+        private static void EnsureAbsolutePath(ref string filePath)
+        {
+            if (!IsPathRooted(filePath))
+            {
+                filePath = Path.GetFullPath(filePath);
+            }
+        }

```

Comment:
Nit pick here... You followed an existing pattern here, but I think I created the wrong pattern of using `ref` originally. Instead, this should be:

```suggestion
        private static string EnsureAbsolutePath(string filePath)
        {
            if (IsPathRooted(filePath))
                return filePath;
            return Path.GetFullPath(filePath);
        }


```

It would be great if you could fix the other one here too. I see it is private so it should be minimal impact.